server/mediaserver.go

```diff
@@ -161,15 +178,13 @@ func gotRTMPStreamHandler(s *LivepeerServer) func(url *url.URL, rtmpStrm stream.
 			minDeposit := big.NewInt(0).Mul(BroadcastPrice, big.NewInt(MinDepositSegmentCount))
 			if deposit.Cmp(minDeposit) < 0 {
 				glog.Errorf("Low deposit (%v) - cannot start broadcast session", deposit)
+				if s.LivepeerNode.MonitorMetrics {
+					monitor.LogStreamCreateFailed(rtmpStrm.GetStreamID(), 0, "LowDeposit")
+				}
 				return ErrBroadcast
 			}
 		}
 
-		// We can only have one concurrent stream for now
-		if len(s.rtmpStreams) > 0 {
```

Comment:
Did you mean to remove this?

Reply:
Yeah - we are doing `ErrAlreadyExists` in line 154 now.

Reply:
Ah -- it would actually allow for several streams under different StreamIDs, since each new connection gets a new RTMP StreamID. Right now, we only allow one stream per node, following the delayed broadcasting proposal. Do we want to change that?